### DISCUSSION TITLE
Mention of Visidata in a longread that got almost 100K views.

### DISCUSSION THREAD

**Discussion by snooppr** [saulpw/visidata#1972] [Show and tell]
Reading the VD [blog](https://www.visidata.org/blog/2021/how-to-contribute/) on the official website, I noticed:
> I love seeing people talking about VisiData, especially their own specific use cases and experiences.

The [longid](https://habr.com/ru/articles/652633/) itself, which mentions Visidata with examples of working on Android.  You can refer to it as a review if you wish.

**Comment by anjakefala** [comment]
Thank you so much for sharing!
